UNETR/BTCV/networks/unetr.py

```
@@ -73,7 +73,7 @@ def __init__(
         if hidden_size % num_heads != 0:
             raise AssertionError("hidden size should be divisible by num_heads.")
 
-        if pos_embed not in ["conv", "perceptron"]:
+        if pos_embed not in ['sincos', 'learnable', 'none']:
             raise KeyError(f"Position embedding layer of type {pos_embed} is not supported.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)